client.go

```diff
@@ -18,8 +18,16 @@ import (
 	"github.com/projectdiscovery/retryabledns/hostsfile"
 )
 
+var internalRangeCheckerInstance *internalRangeChecker
+
 func init() {
 	rand.Seed(time.Now().UnixNano())
+
+	var err error
+	internalRangeCheckerInstance, err = newInternalRangeChecker()
+	if err != nil {
+		panic(err)
```

Comment:
We are parsing hardcoded ipv4/ipv6 addresses, so hopefully, this will never happen, but wouldn't it be better eventually to accept a logger to print the error to (with default to noplogger) or fail silently?

Reply:
it's an error in the implementation side so the panic here ensures that if we ever pass unparsable IP addresses in code this logic would break, hence I added a panic.

Reply:
Though since it's a library ideally we should logic. Will change the code to reflect that